prime-router/src/test/kotlin/FakeReportTests.kt

```diff
@@ -57,14 +58,14 @@ class FakeReportTests {
             ?.findElement("blank_column")
             ?: fail("Lookup failure: blank_column")
         val fakeValue = FakeReport(metadata).buildColumn(blankColumn, rowContext)
-        assertEquals("", fakeValue, "Expected a blank column but received '$fakeValue' instead")
+        assertThat("").isEqualTo(fakeValue)
```

Comment:
Reverse this.  assertThat(actual value).is... like expected.  Do for other similar lines as well.

Reply:
Agreed

Reply:
Ok, got it.

Reply:
Follow up: made the changes.